src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Http/HttpClientHandler/IHttpResponseMessage.cs

```diff
@@ -1,9 +1,11 @@
+using Datadog.Trace.DuckTyping;
+
 namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Http.HttpClientHandler
 {
     /// <summary>
     /// HttpResponseMessage interface for ducktyping
     /// </summary>
-    public interface IHttpResponseMessage
+    public interface IHttpResponseMessage : IDuckType
```

Comment:
Maybe we should revert this in favor of this PR that refactor and handles this scenario already: https://github.com/DataDog/dd-trace-dotnet/pull/1258

Reply:
Done